### ISSUE TITLE
instructions on how to install ijulia kernel for julia 1.2

### ISSUE THREAD

**Issue by shanecandoit** [JuliaLang/IJulia.jl#875]
The given instructions say to enter:

using Pkg
Pkg.add("IJulia")

That did not work for me. Using julia 1.2 these steps worked
1. type "julia" to start julia repl
2. I had to enter '']" to get the "pkg>" prompt.
3. then enter `add "IJulia"`
4. then I could restart my jupyter notebook and julia worked

**Comment by feanor12** [comment]
usually I need to do `build IJulia` to see the kernel in jupyter

**Comment by nilshg** [comment]
This can probably be closed?

**Comment by JamesWrigley** [comment]
Yes, though the above situation could happen if the kernel was deleted and then IJulia was added again since the build step only runs automatically the very first time the package is installed (in a depot?).
